### PR TITLE
feat add support for ORM SchemaIgnoreClasses

### LINKED ISSUE #1454: Add support for `SchemaIgnoreClasses`
I was thinking if we could add `SchemaIgnoreClasses` config (https://github.com/doctrine/orm/pull/9202)

_I checked if I could do it myself but I'm not very familiar with symfony bundles and it seemed too difficult when considered the time I'd have to invest in it_

### PR DESCRIPTION
**Pull request by simPod**
Resolves #1454

### CODE REVIEW

**Comment by simPod** [comment]
Does anyone has any idea how to fix xml support?

File: DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -466,6 +467,9 @@ private function addOrmSection(ArrayNodeDefinition $node): void
                         ->end()
                         ->scalarNode('proxy_dir')->defaultValue('%kernel.cache_dir%/doctrine/orm/Proxies')->end()
                         ->scalarNode('proxy_namespace')->defaultValue('Proxies')->end()
+                        ->arrayNode('schema_ignore_classes')
```
Comment (stof) [inline DependencyInjection/Configuration.php]: this needs a `->fixXmlConfig('schema_ignore_class', 'schema_ignore_classes')` on the parent node (outside the `->children()` block)

File: DependencyInjection/DoctrineExtension.php [hunk 2]
```diff
@@ -643,6 +643,7 @@ protected function loadOrmEntityManager(array $entityManager, ContainerBuilder $
             'setProxyDir' => '%doctrine.orm.proxy_dir%',
             'setProxyNamespace' => '%doctrine.orm.proxy_namespace%',
             'setAutoGenerateProxyClasses' => '%doctrine.orm.auto_generate_proxy_classes%',
+            'setSchemaIgnoreClasses' => '%doctrine.orm.schema_ignore_classes%',
```
Comment (stof) [inline DependencyInjection/DoctrineExtension.php]: Should we really have a single list of ignored classes common for all entity managers ? Shouldn't we have separate list per managers instead ? This is not a configuration mapping to a global state

**Review by stof — CHANGES_REQUESTED** [review]

File: DependencyInjection/DoctrineExtension.php (hunk 2 above)
Comment (simPod) [reply DependencyInjection/DoctrineExtension.php]: I'm not sure it's possible since the `doctrine.orm.configuration` is global and not per EM 🤔

Comment (stof) [reply DependencyInjection/DoctrineExtension.php]: really ? This looks weird to me...

Comment (simPod) [reply DependencyInjection/DoctrineExtension.php]: Is it this? https://github.com/doctrine/DoctrineBundle/blob/4f4387cbbd71a3a5037e44cefa466734499e1e57/DependencyInjection/DoctrineExtension.php#L608

**Comment by ostrolucky** [comment]
There are still some test failures

**Review by ostrolucky — APPROVED** [review]

**Review by dmaicher — APPROVED** [review]

**Comment by dmaicher** [comment]
I tested this on one of my apps and it works as expected :+1: 

Thanks @simPod

**Pull request merged**